setup.py

```diff
@@ -1,70 +1,81 @@
 from setuptools import setup, find_packages
 from setuptools.extension import Extension
+from Cython.Build import cythonize
+from Cython.Distutils import build_ext
+from pysam import get_include as pysam_get_include
+import os
 
-def pip_install(pkg_name):
-    import subprocess
-
-    subprocess.check_call(
-        ["python", "-m", "pip", "install", pkg_name], stdout=subprocess.DEVNULL
-    )
-
-
-try:
-    from Cython.Build import cythonize
-    from Cython.Distutils import build_ext
-except ImportError:
```

Comment:
pyproject.toml should take care of these now, I guess

Reply:
Unfortunately, pyproject.toml does not take care of (hence, doc failling). 
If the part is recovered, will it affect your fix?

Reply:
It won't affect the fix--- please keep it in that case.